### PR TITLE
Clarify the support options for PX4

### PR DESCRIPTION
**Pull request by mrpollo**
* Weekly dev call
* Test Flight team
* Support request
* Calendar
* Code of conduct

### CODE REVIEW

File: en/contribute/README.md [hunk 1]
```diff
@@ -3,7 +3,92 @@
...
+* Test team will [add your PR/Issue to their queue](https://github.com/PX4/Firmware/projects/18)
+
+#### Response times
+
+* Multi-Copter, up to 48hr, typically within 24hr
+* VTOL, Fixed Wing, 4 days, typically 2 days
+
+### Have a problem?
+
+#### Help diagnozing problems
```
Comment (LorenzMeier) [inline en/contribute/README.md]: with "s"?

Comment (mrpollo) [reply en/contribute/README.md]: just resolved, thanks

**Comment by mrpollo** [comment]
to whoever approves and merges, please SQUASH my 2 commits

File: en/contribute/README.md [hunk 2]
```diff
@@ -3,7 +3,92 @@
...
+
+
+{% raw %}
+<iframe src="https://calendar.google.com/calendar/embed?title=Dronecode%20Calendar&amp;mode=WEEK&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=linuxfoundation.org_g21tvam24m7pm7jhev01bvlqh8%40group.calendar.google.com&amp;color=%23691426&amp;ctz=Europe%2FZurich" style="border-width:0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+{% endraw %}
+
+
+## Contributing
+
+### [Code of Conduct](https://github.com/PX4/Firmware/blob/master/CODE_OF_CONDUCT.md)
```
Comment (hamishwillee) [inline en/contribute/README.md]: Please make this heading not a link.

File: en/contribute/README.md [hunk 3]
```diff
@@ -3,7 +3,92 @@
...
+
+**Note:** calendar defaults to CET.
+
+
+{% raw %}
+<iframe src="https://calendar.google.com/calendar/embed?title=Dronecode%20Calendar&amp;mode=WEEK&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=linuxfoundation.org_g21tvam24m7pm7jhev01bvlqh8%40group.calendar.google.com&amp;color=%23691426&amp;ctz=Europe%2FZurich" style="border-width:0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+{% endraw %}
+
+
+## Contributing
```
Comment (hamishwillee) [inline en/contribute/README.md]: Remove this heading. MOve the Code of conduct heading up to level 2

**Comment by dagar** [comment]
Can we get a dedicated page for test flights and link to it here?
I'd then like to expand it to include the vehicles, optional equipment, and types of tests available.

**Comment by hamishwillee** [comment]
@dagar EDITED: OK, I have created #431
Can you work with @PX4/testflights to update this as you will?

**Comment by hamishwillee** [comment]
@mrpollo Merged. Didn't squash your commits - github didn't give me the option and I don't know how to do manually. Not particularly concerned about clean commit pattern for docs, though I guess it would be nice.

**Pull request merged**